### PR TITLE
Add Materialize and upgrade Ruby to 2.3.1

### PR DESCRIPTION
**Pull request by foxycoder**
- Added `materialize-sass`
- Upgraded SASS and Compass depencies
- Upgraded Ruby version from 2.2.4 to 2.3.1

After this is merged, everyone should do:

```bash
# Set the new Ruby version
rbenv local 2.3.1 

# Install the new dependencies
bundle install
```

### CODE REVIEW

File: app/assets/stylesheets/application.css.sass [hunk 1]
```diff
@@ -1,10 +1,2 @@
...
-@import "bootstrap-sass/assets/stylesheets/bootstrap-sprockets"
-@import "bootstrap-sass/assets/stylesheets/bootstrap"
-@import "devise_bootstrap_views"
-@import "rails_bootstrap_forms"
-@import "froala_editor.min"
-@import "froala_style.min"
+@import "materialize"
 @import "font-awesome"
-@import "base/*"
-@import "components/*"
```
Comment (miriamtocino) [inline app/assets/stylesheets/application.css.sass]: Not sure, but don't we need `base/` and `/components` to be there still?

Comment (foxycoder) [inline app/assets/stylesheets/application.css.sass]: Nope.

Comment (foxycoder) [inline app/assets/stylesheets/application.css.sass]: We stripped all styling (since that depends on bootstrap)

Comment (foxycoder) [inline app/assets/stylesheets/application.css.sass]: Oh, and Admin still uses bootstrap, but that has it's own `admin.css` and `admin.js` manifests.

Comment (miriamtocino) [inline app/assets/stylesheets/application.css.sass]: Awesome!

**Pull request merged**